### PR TITLE
Link "virtual machine" and "cloud native"

### PR DESCRIPTION
**Pull request by HenokB**
Signed-off-by: Henok Biadglign Ademtew <46082799+HenokB@users.noreply.github.com>

### CODE REVIEW

File: content/en/nodes.md [hunk 1]
```diff
@@ -23,6 +23,6 @@ Thus, nodes run apps or processes as part of a group forming a [cluster](/cluste
 ## How it helps
 
 A node gives you a distinct unit of compute (memory, CPU, network) that you can assign to a cluster. 
-In a cloud native platform or app a node represents a single unit that can perform work. 
+In a [cloud native](/cloud-native-apps/) platform or app a node represents a single unit that can perform work.
```
Comment (iamNoah1) [inline content/en/nodes.md:26]: I would rather link this to cloud native technology than to cloud native apps, what do you think @jihoon-seo @seokho-son @CathPag ?

**Review by iamNoah1 — COMMENTED** [review]
Thanks for the contribution @HenokB. I have one little suggestion :) Let's see what the others think.

File: content/en/nodes.md (hunk 1 above)
Comment (seokho-son) [reply content/en/nodes.md:26]: @iamNoah1 
I also prefer to link to cloud native technology. (maybe related description needs to be enhanced accordingly)

Comment (HenokB) [reply content/en/nodes.md:26]: Thanks @iamNoah1 and @seokho-son I will change it now.

**Review by CathPag — APPROVED** [review]
LGTM

**Review by iamNoah1 — APPROVED** [review]
thanks @HenokB 
lgtm

**Pull request merged**